build.gradle

```diff
@@ -21,15 +20,11 @@ minecraft {
 
 dependencies {
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:${project.yarn_mappings}"
+	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
+	modImplementation "net.fabricmc.fabric-api:fabric-api:${fabric_api_version}"
 
-	modCompileOnly "net.fabricmc.fabric-api:fabric-api:${fabric_api_version}"
-	modRuntime "net.fabricmc.fabric-api:fabric-api:${fabric_api_version}"
-	modImplementation ("io.github.cottonmc:LibGui:${project.libgui_version}") {
-		exclude group: 'net.fabricmc.fabric-api'
-	}
-
+	modImplementation "io.github.cottonmc:LibGui:${project.libgui_version}"
```

Comment:
Revert these. `modCompileOnly` is combined with `modRuntime` in order to prevent downstream dependents from pulling in potentially old Fabric API modules.

Reply:
There’s another more pretty way to do this; it is by either excluding the module from the naughty dependency or by using Gradle 6 rich versions https://docs.gradle.org/current/userguide/rich_versions.html